docs/guides/postgres/custom-versions/extensions.md

```
@@ -0,0 +1,412 @@
+kubectl delete -n demo secret pg-extensions-config
+kubectl delete ns demo
+```
+
+## Next Steps
+
+- Learn how to [run PostgreSQL with a custom configuration file](/docs/guides/postgres/configuration/using-config-file.md).
+- Set up a [Highly Available PostgreSQL cluster](/docs/guides/postgres/clustering/ha_cluster.md) — extensions created on the primary replicate to the standbys.
+- Monitor your PostgreSQL database with KubeDB using [built-in Prometheus](/docs/guides/postgres/monitoring/using-builtin-prometheus.md).
+- Detail concepts of [PostgresVersion object](/docs/guides/postgres/concepts/catalog.md).
+- Want to hack on KubeDB? Check our [contribution guidelines](/docs/CONTRIBUTING.md).
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)